hera_mc/tests/test_geo_location.py

```diff
@@ -39,6 +39,15 @@ def test_geo_sysdef():
     assert int(a['HH123']['E']) == 540945
 
 
+def test_geo_regions():
```

Comment:
It'd be good to see more tests. I'd try having lists with multiple entries. I'd try all the different types you mentioned in the PR, e.g. `123, '123', 'HH123', 'A123'`

Reply:
Added